index.js

```diff
@@ -1,7 +1,30 @@
 // Iteration 1: Names and Input
 
+var hacker1 = { firstName: 'Jeremy', lastName: 'Couderc' };
+var message = 'The drivers name is ' + hacker1.firstName + ' ' + hacker1.lastName + '!!';
```

Comment:
We usually avoid `var` to define our variables now, because they can escape scope and that can be troublesome for some applications